deploy/manifests/dev-v2/us-east-2/cluster/storetheindex/.sops.yaml

```diff
@@ -0,0 +1,4 @@
+creation_rules:
+  - path_regex: .*
+    encrypted_regex: '^(data|stringData)$'
+    kms: 'arn:aws:kms:us-east-2:407967248065:alias/sti_flux'
```

Comment:
This is reusing the same KMS key right? Is that the plan for dev v2 going forward?

General question: where do terraforms that create such resources live? will they be managed centrally or remain as they are in application repo?

Reply:
It is not the plan going forward I actually missed this thank you for pointing it out. I have to add key creation to the template, then re-encrypt the key. Please see https://github.com/filecoin-project/gitops-root/issues/198 for tracking. 

Long term strategy still needs to be developed with you and the team.

Reply:
It would be done using terraform